app/src/pages/settings/UsersTable.tsx

```diff
@@ -71,7 +80,30 @@ export function UsersTable({ query }: { query: UsersTable_users$key }) {
       },
       {
         header: "role",
-        accessorKey: "roleName",
+        accessorKey: "role",
+        cell: ({ row, getValue }) => {
+          return (
+            <RolePicker
+              includeLabel={false}
+              onChange={(key) => {
+                if (key === row.original.role) {
+                  return;
+                }
```

Comment:
we shouldn't let admin@localhost's role to be changed. It causes the server to no longer be able to be booted. This should also be blocked on the server @RogerHYang

Reply:
oh yeahi had a block on this for the delete, will do the same here